src/reducers/query.jsx

```diff
@@ -998,6 +1009,8 @@ export default ( state = defaultQuery, action ) => {
     validateDateInterval( newState )
   }
 
+  // remove any filter keys with empty array
+  pruneEmptyFilters( newState )
```

Comment:
Wait... what happens in this sequence

1. Filter on `90210`
1. Filter on State = `Kansas`

Both filters get dropped?

Reply:
doesn't seem like its getting dropped from the query reducer
![filter](https://user-images.githubusercontent.com/1038306/87693058-7f99ee80-c75a-11ea-9a6a-56db94073762.gif)

Reply:
```
    zip_code: [
      '90210'
    ],
    state: [
      'KS'
    ]
```

So the filter arrays have values, but the counts are zero.. gotcha

Reply:
Thanks for the clarification!